lib/galaxy/job_metrics/instrumenters/cgroup.py

```diff
@@ -17,7 +18,27 @@
 
 log = logging.getLogger(__name__)
 
+VALID_VERSIONS = ("auto", "1", "2")
+DEFAULT_PARAMS = (
+    # cgroupsv1 - probably more than useful but don't remove any for legacy reasons
```

Comment:
I don't understand the comment ... are they not useful ?

Reply:
Things like `memory.soft_limit_in_bytes` are probably not useful, I just didn't want to change the default set that has been captured since the plugin was written.

Reply:
> The phrase 'more than useful' is an acceptable phrase in written English. It is used when something is not only useful, but more than useful. For example, "This book has been more than useful in helping me understand the subject.". 

TIL

Reply:
Ah sorry for the confusion, that's not what I was aiming for, it'd be clearer if written as "this is probably more params than are useful to collect ..."